### ISSUE TITLE
Dynamic Mode only working on one side on a super ultra-wide screen

### ISSUE THREAD

**Issue by ToggoStar** [torchgm/RoundedTB#98]
Hello!

First off: RoundedTB is an awesome idea and looks really promising, thank you for the work!

Now the issue: On my super-ultra wide monitor, dynamic mode only seems to work properly on the right side. On the left side, the taskbar appears as usual. Any ideas what might be causing this?

![Screenshot 2021-11-13 215521](https://user-images.githubusercontent.com/19649347/141658825-59b0eb84-a42a-4434-85ab-10d9216a45d4.png)

I'm using the version 3.1B.

Cheers

**Comment by torchgm** [comment]
P3.1 B was *supposed* to have a fix for this, but for now go ahead and just set the taskbar icons to the left then back to the centre. I'll doubly-make-sure this fix is implemented before R3.1 actually releases.

**Comment by torchgm** [comment]
Could you provide the config file? Help & About > Debug > Open log file

**Comment by mchlbataller** [comment]
hi, this is the config file if it helps.

I have the same issue as him so I hope it doesn't matter whoever you got the config file from.

```json
{
  "CornerRadius": 6,
  "MarginBasic": 0,
  "MarginBottom": 0,
  "MarginLeft": 0,
  "MarginRight": 0,
  "MarginTop": 0,
  "IsDynamic": true,
  "IsCentred": false,
  "ShowTray": false,
  "CompositionCompat": false,
  "IsNotFirstLaunch": true
}
```

**Comment by ToggoStar** [comment]
> 
> 
> Could you provide the config file? Help & About > Debug > Open log file

Here's my config:

{
  "Version": 1,
  "CornerRadius": 7,
  "MarginBasic": 0,
  "MarginBottom": 0,
  "MarginLeft": 0,
  "MarginRight": 0,
  "MarginTop": 0,
  "IsDynamic": true,
  "IsCentred": false,
  "IsWindows11": true,
  "ShowTray": false,
  "CompositionCompat": false,
  "IsNotFirstLaunch": true,
  "FillOnMaximise": false,
  "FillOnTaskSwitch": false,
  "ShowTrayOnHover": true
}

Thanks for looking into it!

**Comment by slunat** [comment]
Was just coming to report I have the same issue. Setting to left and back to center fixed it for me aswell.

**Comment by koddajr** [comment]
Yep, align left and then center fixed it

**Comment by redfellow** [comment]
Fresh Win 11 install, still happens if you never toggled task bar position to begin with.

**Comment by Erisa** [comment]
Yeah it seems to happen in almost all cases, hopefully will be resolved in the next update.

**Comment by hieunhoo** [comment]
if i adjust to left then back to center it solves the problem but gives rise to the 'weather' not showing up and i use it a lot again. Don't know if there is a way to solve this problem.

**Comment by JaneValleyhasBtaken** [comment]
OK, so I am currently on the 22623.857, and somehow the dynamic mode just work on the both side of monitor. 
Like, Microsoft, what are you playing at.

**Comment by eglish** [comment]
Reporting in that I just did a fresh install of Win11.  Current version: Microsoft Windows [Version 10.0.22000.1516].  This is still an issue

**Comment by GlassWolfLH** [comment]
I fixed this issue by disabling dynamic mode, opening the taskbar settings window, re-enabling dynamic mode, then in taskbar settings, open taskbar behaviors, and on "taskbar alignment" change it from center to left, then back to center, and all should work properly.

**Comment by rollingmoai** [comment]
Unfortunately, dynamic mode is broken in the latest stable version of Windows (`22631.2338`).
- https://github.com/RoundedTB/RoundedTB/issues/320
![image](https://github.com/RoundedTB/RoundedTB/assets/75471378/1fcec63f-bed2-4130-b2f7-953576705340)

Also see:
- https://github.com/RoundedTB/RoundedTB/issues/330
